index.html

```diff
@@ -2,14 +2,128 @@
+            <div class="hero-img-container">
+                <img class="hero-img" src="./assets/heroimge.jpg" alt="" />
+                <div class="hero-content">
+                    <p class="hero-desc">
+                        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos rem
+                        sit eaque beatae nisi cum, maxime fugiat! Atque ipsum debitis rem
+                        officia facilis ab optio, sapiente quasi. Aut, ut explicabo!
+                    </p>
+                    <button class="secondary-btn hero-btn">Explore</button>
+                    <button class="primary-btn hero-btn">Shop Now</button>
+                </div>
+            </div>
+            <!-- description about the website -->
+        </heroImg>
+
+        <funnelImg>
+            <h1 class="align-center txt-clr-primary txt-fs-mont">
+                <u>Shop By The Most Favorited Catagories</u>
+            </h1>
+
+            <div class="category-img-container">
+                <img class="category-img" src="./assets/trending.jpg" alt="" />
+                <img class="category-img" src="./assets/bestsellers.jpg" alt="" />
+                <img class="category-img" src="./assets/latest.jpg" alt="" />
+                <img class="category-img" src="./assets/valentine's.jpg" alt="" />
+            </div>
+            <!-- mini images of 4 for 
+            1. pocket friendly store
+            2. bestsellers of the month
+            3. latest in tech
+            4. valentine's day special -->
+        </funnelImg>
+
+        <content>
+            <!-- 
+    5-6 images:-
+         trending accessories 
+         trending accessories 
+         trending accessories 
+         trending accessories 
+         trending accessories 
+         trending accessories 
+    -->
+            <!-- <h1 class="align-center txt-clr-primary txt-fs-mont">
+                <u>Trending Accessories</u>
+            </h1>
+
+            <div class="trending-accessories-img-container">
+                <img class="trending-accessories-img" src="./assets/accessory/31bBMCBWCVL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/519uMzMXJVL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/51spbABphFL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61+Lfxgog+S._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61DKcGr12OL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61gscZYmaoL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61vtLhO6fDL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/71lVwl3q-kL._AC_UL320_.jpg" alt="" />
+            </div>
```

Comment:
If you not gonna use this commented part, clean up.

Reply:
I need them. 👍🏻